teps/0114-custom-tasks-beta.md

```diff
@@ -0,0 +1,262 @@
+---
+status: implementable
+title: Custom Tasks Beta
+creation-date: '2022-07-12'
+last-updated: '2022-07-12'
+authors:
+- '@jerop'
+see-also:
+- TEP-0002
+- TEP-0061
+- TEP-0069
+- TEP-0071
+- TEP-0096
+- TEP-0105 
+---
+
+# TEP-0114: Custom Tasks Beta
+
+<!-- toc -->
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+  - [Proposal](#proposal)
+    - [Required](#required)
+      - [API Changes](#api-changes)
+        - [v1alpha1 to v1beta1 + Run to CustomRun](#v1alpha1-to-v1beta1--run-to-customrun)
+        - [Embedded Specifications](#embedded-specifications)
+        - [Feature Gates](#feature-gates)
+      - [Documentation](#documentation)
+      - [Testing](#testing)
+    - [Optional](#optional)
+      - [Authoring Custom Tasks](#authoring-custom-tasks)
+      - [Sharing Custom Tasks](#sharing-custom-tasks)
+        - [Custom Tasks in Tekton Hub](#custom-tasks-in-tekton-hub)
+  - [References](#references)
+<!-- /toc -->
+
+## Summary
+
+The aim of this TEP is to promote `Custom Tasks`, which are used to extensibility in Tekton Pipelines, from alpha to 
+beta.
+
+## Motivation
+
+[TEP-0002: Custom Tasks][tep-0002] introduced `Custom Tasks` which allow users to extend the functionality of Tekton 
+Pipelines. Users implement controllers that watch for `Runs` that implement their types of `Custom Tasks`. 
+
+Today, `Runs` is in `v1alpha1`; this was one of the blockers for removing `v1alpha1` completely in [TEP-0105][tep-0105].
```

Comment:
Note that we don't need to completely remove `v1alpha1`. I think the TEP-105 is mainly "remove a set of objects from v1alpha1" —aka all the deprecated one, and all the one that got *promoted* into v1beta1.

Nothing should prevent us from adding new objects in `v1alpha1` in the future. Each "object" has its own version lifecycle.

Reply:
makes sense, updated the proposal - thanks @vdemeester!